sqe-http-api/DTOs/Artefact.cs

```diff
@@ -11,16 +13,46 @@ public class ArtefactDTO
         public uint editionId { get; set; }
         public string imagedObjectId {get; set;}
         public string name { get; set; }
-
-        // Bronson: PolygonDTO already has a mask and matrix. Are you sure this is correct?
         public PolygonDTO mask { get; set; }
-        public string transformMatrix { get; set; }
         public uint zOrder { get; set; }
         public ArtefactSide side { get; set; }
 
         public enum ArtefactSide { recto, verso}
     }
 
+    public static class ArtefactDTOTransform
+    {
+        public static ArtefactDTO QueryArtefactToArtefactDTO(ArtefactsOfEditionQuery.Result artefact, uint editionId)
+        {
+            return new ArtefactDTO()
+            {
+                id = artefact.artefact_id,
+                editionId = editionId,
+                mask = new PolygonDTO()
+                {
+                    mask = artefact.mask,
+                    transformMatrix = ""
+                },
+
+                imagedObjectId = ImagedObjectIdFormat.Serialize(artefact.institution, artefact.catalog_number_1, artefact.catalog_number_2),
+
+                name = artefact.name,
+                side = artefact.catalog_side == 0 ? ArtefactDTO.ArtefactSide.recto : ArtefactDTO.ArtefactSide.verso, 
+                zOrder = 0,
+            };
+        }
+        
+        public static ArtefactListDTO QueryArtefactListToArtefactListDTO(List<ArtefactsOfEditionQuery.Result> artefacts, uint editionId)
```

Comment:
First, I think we placed all the ModelToDTO methods in services and not the DTOs themselves. I think the DTOs should n't be aware of the models.
Second, the repositories should return models, not query result objects

Reply:
That is fine with me.  This particular code is used by both the ImagedObjectService and the ArtefactService, so I guess it needs to live outside those.  I created a file ArtefactDTOTransformService.cs, but perhaps there is a better place to put such multi-use transformers.

I have to run to a parent-teacher meeting right now.  Can we talk on slack about the issue of these intermediary models between the repositories and the services.

Reply:
We have a bunch of conversions like this, let's all put them in the same place. Note that this is *not* a service, but rather a static class (a class that only has static methods). It is not injectable, and isn't registered anywhere.

Reply:
Yes, you are absolutely right, we will have many of these class "transformers", which will all be static (some may even build upon others with inheritance). So should I put it in the "Helpers" folder, or should I create a new folder (and namespace) with these static class transformation methods?  What would I call it?

Reply:
I think the Helpers folder is great for this.

Reply:
Sounds good to me.  I will do it today.